npm_test.go

```diff
@@ -196,7 +207,7 @@ func createNpmProject(t *testing.T, dir string) string {
 	return packageJson
 }
 
-func validateNpmInstall(t *testing.T, npmTestParams npmTestParams) {
+func validateNpmInstall(t *testing.T, npmTestParams npmTestParams, isNpm7 bool) {
```

Comment:
The ```isNpm7``` argument seem to be unused.
The same goes for the ```validateNpmPackInstall``` function.

Reply:
The argument is there to comply with the signature of the [validationFunc](https://github.com/RobiNino/jfrog-cli/blob/c9ca9e3b38aa09304058ff8014924b62f9748cbe/npm_test.go#L39)

Reply:
Got it. NP.